specification/communication/data-plane/SipRouting/preview/2024-11-15-preview/communicationservicessiprouting.json

```diff
@@ -365,6 +376,18 @@
         "enabled": {
           "description": "Enabled flag",
           "$ref": "#/definitions/Enabled"
+        },
+        "stepOutOfSignaling": {
+          "type": "boolean",
+          "description": "StepOutOfSignaling enabled flag."
+        },
+        "privacyHeader": {
+          "$ref": "#/definitions/PrivacyHeader",
+          "description": "SIP Privacy header."
+        },
+        "ipAddressVersion": {
+          "$ref": "#/definitions/IpAddressVersion",
+          "description": "IP address version used by the trunk."
```

Comment:
Do we want to mention the default value in case that the PATCH is a create operation?